### ISSUE TITLE
Constructing a Paginator triggers creation of unexpected page1 pages.

### ISSUE THREAD

**Issue by dimo414** [spf13/hugo#1319]
Demo:

```
$ mkdir demo

$ cd demo/

$ hugo new site .

$ mkdir themes

$ cp -R ../hugo/themes/beg/ themes/

$ cp themes/beg/config.yaml .

$ rm config.toml

$ hugo new post/demo.md
C:\cygwin64\tmp\demo\content\post\demo.md created

$  hugo --buildDrafts -t=beg
1 of 1 draft rendered
0 future content
1 pages created
0 paginator pages created
0 tags created
in 49 ms

$ find public/ -name *page*

$ sed -i '1i{{ $paginator := .Paginate .Data.Pages }}' themes/beg/layouts/index.html

$ hugo --buildDrafts -t=beg
1 of 1 draft rendered
0 future content
1 pages created
1 paginator pages created
0 tags created
in 46 ms

$ find public/ -name *page*
public/page1

$ cat public/page1/index.html
<!DOCTYPE html><html><head><link rel="canonical" href="http://example.com/"/><meta http-equiv="content-type" content="text/html; charset=utf-8" /><meta http-equiv="refresh" content="0;url=http://example.com/" /></head></html>
```

As you can see, there was not previously a `page1` page, but calling `.Paginate` to construct a paginator starts creating these unexpected `page1` files.

I don't see a mention of these pages in the [documentation](http://gohugo.io/extras/pagination/), other than:

```
[SECTION/TAXONOMY/BLANK]/page/1/index.html => redirect to  [SECTION/TAXONOMY/BLANK]/index.html
```

Perhaps there's a typo, and it should be generating `page/1` not `page1`?  Even so, I would prefer if these files were only generated once the pages *actually* need to be paginated.  Many lists will only ever be a single page by design, and don't need these `page1` pages.

**Comment by bep** [comment]
It should be `/page/1` - what OS and Hugo version are you running?

That `/page/1` gets generated (it's just a small redirect) even if there is only 1 page is by design. Main motivation is to support "inifinite scroll" libraries, but it is also just, right.

**Comment by dimo414** [comment]
I'm on Cygwin, running Hugo 0.14.  Seems like a Windows specific issue, I was able to replicate it in the Windows Command Prompt (`$` for Cygwin, `>` for cmd):

```
$ rm -rf public/*

> hugo.exe --buildDrafts -t=beg
1 of 1 draft rendered
0 future content
1 pages created
1 paginator pages created
0 tags created
in 29 ms

$ find public/ -name *page*
public/page1
```

**Comment by bep** [comment]
I'm not able to reproduce this on Windows 8.1 (nor OSX). Hmm ...

**Comment by dimo414** [comment]
To clarify, I'm running Windows 7.  I'll try to find another Windows box and see if I can replicate it.

**Comment by dimo414** [comment]
I just noticed the logs printed from `hugo server -v` also have a miss-escaping, e.g.:

    INFO: 2015/08/11 File System Event: ["E:\\Workspace\\blog\\data\\titles.toml": CREATE "E:\\Workspace\\blog\\data\\titles.toml": WRITE]

In case that helps find the cause of the original problem.

**Comment by luixxiul** [comment]
> That /page/1 gets generated (it's just a small redirect) even if there is only 1 page is by design.

Is there any option to disable it?

**Comment by bep** [comment]
No.

**Comment by dimo414** [comment]
Another datapoint; it's possible my Cygwin install is mangled in some way, I've run into a problem with [Mercurial thinking I'm not on Windows](http://stackoverflow.com/q/33359891/113632) - is it possible Hugo has some similar OS check?

**Comment by bep** [comment]
@dimo414 yes.

**Comment by dimo414** [comment]
Ok. To rephrase, could you point me that way? Or does this shed any light on the issue? Even if my setup is mangled, this failure mode is strange and I'd like to try to hunt it down.

**Comment by dimo414** [comment]
This issue appears to be resolved in Hugo 0.15.  Perhaps it was a bug in Go that was addressed?

```
$ hugo_0.14_windows_amd64.exe --buildDrafts -t=beg
0 draft content
0 future content
1 pages created
1 paginator pages created
0 tags created
0 categories created
in 21 ms

$ find public/ -name *page*
public/page1

$ rm -rf public/

$ hugo_0.15_windows_amd64.exe --buildDrafts -t=beg
0 draft content
0 future content
1 pages created
1 paginator pages created
0 tags created
0 categories created
in 21 ms

$ find public/ -name *page*
public/page
```
